src/simulation/dynamics/DynOutput/boreAngCalc/boreAngCalc.cpp

```diff
@@ -86,30 +86,30 @@ void BoreAngCalc::ReadInputs()
     @return void
 */
 void BoreAngCalc::computeAxisPoint()
-{
-    double dcm_BN[3][3];                /*!< dcm, inertial to body frame */
-    double dcm_PoN[3][3];               /*!< dcm, inertial to Point frame */
-    double dcm_BPo[3][3];               /*!< dcm, Point to body frame */
-    double relPosVector[3];
-    double relVelVector[3];
-    double secPointVector[3];
-    double primPointVector[3];
-    
-    MRP2C(localState.sigma_BN, dcm_BN);
-    v3Subtract(localPlanet.PositionVector, localState.r_BN_N, relPosVector);
-    v3Subtract(localPlanet.VelocityVector, localState.v_BN_N, relVelVector);
-    v3Cross(relPosVector, relVelVector, secPointVector);
-    v3Normalize(secPointVector, secPointVector);
-    v3Normalize(relPosVector, primPointVector);
-    v3Copy(primPointVector, &(dcm_PoN[0][0]));
-    v3Cross(primPointVector, secPointVector, &(dcm_PoN[2][0]));
-    v3Normalize(&(dcm_PoN[2][0]), &(dcm_PoN[2][0]));
-    v3Cross(&(dcm_PoN[2][0]), &(dcm_PoN[0][0]),
-            &(dcm_PoN[1][0]));
-    m33MultM33t(dcm_BN, dcm_PoN, dcm_BPo);
-    Eigen::MatrixXd vecBore_B = Eigen::Map<Eigen::MatrixXd>(boreVec_B, 3, 1);
-    m33tMultV3(dcm_BPo, vecBore_B.data(), boreVecPoint);
-    
+{             
+    // Convert planet and body data to Eigen variables
+    Eigen::Vector3d r_BN_N = cArray2EigenVector3d(this->localState.r_BN_N);
+    Eigen::Vector3d v_BN_N = cArray2EigenVector3d(this->localState.v_BN_N);
+    Eigen::Vector3d planetPositionVector = cArray2EigenVector3d(this->localPlanet.PositionVector);
```

Comment:
`planetPositionVector` should be `r_PN_N`

Reply:
Done